__tests__/mocks/mock-fs.ts

```diff
@@ -3,17 +3,28 @@ import * as path from 'path';
 
 export const drive = process.platform === 'win32' ? 'C:' : '/';
 export const root = path.join(drive, 'Users', 'tester', 'Code', 'app');
+export const tmp = path.join(drive, 'tmp');
 export const numberOfFiles = 14;
 
 const staticDir = path.join(__dirname, '../../static');
 const staticContent: Record<string, Record<string, any>> = {};
 staticContent[staticDir] = {};
 
+const vendorDir = path.join(__dirname, '../../vendor');
+const vendorContent: Record<string, Record<string, any>> = {};
+vendorContent[vendorDir] = {};
+
 fs.readdirSync(staticDir)
   .forEach((file) => {
     staticContent[staticDir][file] = fs.readFileSync(path.join(staticDir, file), 'utf-8');
   });
 
+  fs.readdirSync(vendorDir)
+  .forEach((file) => {
```

Comment:
Stupid nit: Indents here!

(Feel free to ignore that, I can just slap on prettier or something similar later)